packages/docusaurus-theme-classic/src/theme/MDXPage/index.tsx

```diff
@@ -66,6 +76,17 @@ export default function MDXPage(props: Props): JSX.Element {
               </div>
             )}
           </div>
+          {canDisplayEditMetaRow && (
+            <EditMetaRow
+              className={clsx(
+                'margin-top--sm',
+                ThemeClassNames.blog.blogFooterEditMetaRow,
```

Comment:
Should it be removed or be added in `docusaurus-theme-common/ThemeClassNames.ts` ?

Reply:
Yes we need a `ThemeClassNames.pages.pageFooterEditMetaRow`